settings.lua

```diff
@@ -3276,6 +3296,142 @@ function ForcedToStopSetting:init(vehicle)
 	self:set(false)
 end
 
+---
+
+---@class ReverseSpeedSetting : SpeedSetting
+ReverseSpeedSetting = CpObject(SpeedSetting)
+function ReverseSpeedSetting:init(vehicle)
+	SpeedSetting.init(self, 'reverseSpeed','COURSEPLAY_SPEED_REVERSING', 'COURSEPLAY_SPEED_REVERSING', vehicle,3,(vehicle:getCruiseControlMaxSpeed() or 60)) 
+	self:set(6)
+end
+
+function ReverseSpeedSetting:onChange()
+	self.vehicle.cp.speeds.reverse = self:get()
```

Comment:
Guess the intention is that we'll gradually switch to reverseSpeedSetting from vehicle.cp.speeds.reverse? Same for the others?

Reply:
Yes that was my idea.